app/components/form/save-with-history.hbs

```diff
@@ -17,7 +17,11 @@
   <Modal.Footer>
     <AuToolbar as |Group|>
       <Group>
-        <AuButton {{on "click" @save}}>
+        <AuButton
+          {{on "click" @save}}
+          @loading={{@isSaving}}
+          @loadingMessage="Aan het opslaan"
+        >
```

Comment:
Not PR related but added this as when you are waiting on the save nothing is shown to the user